modelci/types/trtis_objects.py

```diff
@@ -566,7 +566,7 @@ class ModelOptimizationPolicy(betterproto.Message):
     optimized and prioritized by the backend framework when@@   it is loaded.@@
     """
 
-    # @@  .. cpp:var:: Graph graph@@@@     The graph optimization setting for the
+    # @@  .. cpp:var:: Graph graph.py@@@@     The graph.py optimization setting for the
```

Comment:
Please revert changes in this file. Seems like they are irrelevant

Reply:
> Please revert changes in this file. Seems like they are irrelevant

OK!